lib/ex_doc/formatter/html/templates.ex

```diff
@@ -30,6 +30,13 @@ defmodule ExDoc.Formatter.HTML.Templates do
     nil
   end
 
+  @doc """
+  Indents specs after line breaks.
+  """
+  def indent_spec(string) do
+    String.replace(string, "\n", "\n    ")
```

Comment:
Great find. :heart: I think we should potentially indent when we convert the spec to string, and not inside the template. Can you please investigate? We would also need a test. Thank you.

Reply:
done. There is one issue though is that the lines after the first one may exceed 80 chars, one solution is to set the width to 76 instead of 80.